guides/main_concepts/relevancy.md

```diff
@@ -109,49 +101,53 @@ The `word position` rule orders by ascending matching word's index number.
 
 ### Exactness
 
-`Knight Moves` is displayed before `Knights of Badassdom` because `Knight` is a complete word, the same as the search query. While with `Knights`, the search query is a prefix of the query word.
+`Knight Moves` is displayed before `Knights of Badassdom`. `Knight` is exactly the same as the search query `Knight` whereas there is a letter of difference between `Knights` and the search query `Knight`.
 
 :::
 
 ::::
 
 ## Order of the rules
 
-By default, the built-in rules are in a specific order that MeiliSearch consider the most suitable for common needs.
+By default, the built-in rules are executed in the following order to meet most standard needs.
 
 ```json
 ["typo", "words", "proximity", "attribute", "wordsPosition", "exactness"]
 ```
 
-Depending on your needs, you might want to change the order of the rules.
-You can use the [settings route](/references/ranking_rules.md#update-ranking-rules) of your index to do so.
+Depending on your needs, you might want to change this order of importance. To do so, you can use the [settings route](/references/ranking_rules.md#update-ranking-rules) of your index.
 
 ## Adding your rules
 
-New rules can be added to the existing list at any time and anywhere in the existing order.
+New rules can be added to the existing list at any time and placed anywhere in the sequence.
+
+A custom rule allows you to create an ascending or descending sorting rule on a given attribute.
 
-A custom rule lets you create a descending or ascending sorting rule on a given attribute.<br>
-To create a rule, you need to communicate the attribute on which the rule is created and the order in which it will be sorted: `asc(attribute_name)` or `desc(attribute_name)`.
+To add your own ranking rule, you have to communicate either `asc` for ascending order or `dsc` for descending order followed by the field name in brackets.
```

Comment:
Its either round brackets or parenthesis. Brackets is this `[ ]` 
```suggestion
To add your own ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field's attribute between round brackets.
```

Reply:
I can't commit this suggestion.